android/app/src/main/AndroidManifest.xml

```diff
@@ -31,7 +31,8 @@
             </intent-filter>
 
             <!-- Deep links -->
-            <intent-filter>
+            <meta-data android:name="flutter_deeplinking_enabled" android:value="true" />
```

Comment:
Should we add the iOS counterpart of this change on Info.plist? Did not having this broke behavior on Android?

Reply:
Yes agree, I have added it now.